packages/sdk/src/HopBridge.ts

```diff
@@ -457,39 +458,11 @@ class HopBridge extends Base {
       estimatedReceived = BigNumber.from(0)
     }
 
-    let requiredLiquidity = hTokenAmount
-    const bondableChains = ['optimism', 'arbitrum']
-    const isBondableChain = bondableChains.includes(sourceChain.slug)
-    const includePendingAmount =
-      destinationChain.equals(Chain.Ethereum) && isBondableChain
-
-    if (includePendingAmount) {
-      const bridgeContract = await this.getBridgeContract(sourceChain)
-      const pendingAmount = await bridgeContract.pendingAmountForChainId(
-        destinationChain.chainId
-      )
-      let pendingAmounts = BigNumber.from(0)
-      for (const chain of bondableChains) {
-        const exists = this.getL2BridgeAddress(this.tokenSymbol, chain)
-        if (!exists) {
-          continue
-        }
-        const bridge = await this.getBridgeContract(chain)
-        const pendingAmount = await bridge.pendingAmountForChainId(
-          destinationChain.chainId
-        )
-        pendingAmounts = pendingAmounts.add(pendingAmount)
-      }
-      requiredLiquidity = requiredLiquidity
-        .add(pendingAmounts)
-        .add(hTokenAmount) // account for transfer root bonds
-    }
-
     return {
       amountOut,
       rate,
       priceImpact,
-      requiredLiquidity,
+      requiredLiquidity: hTokenAmount,
```

Comment:
Will this affect [this line](https://github.com/hop-protocol/hop/blob/8eee9b4c3c594ea65a45489984f128e7d830ba9b/packages/frontend/src/pages/Send/Send.tsx#L431) downstream?

Reply:
requiredLiquidity is simply the amount the bonder will need to bond (in this case it's the hToken amount output from swap)

Reply:
Understood. I was reading this incorrectly. Makes sense & will resolve this now.